launch_xml/test/launch_xml/test_arg.py

```diff
@@ -38,6 +38,27 @@ def test_arg():
     assert 'something' == arg.description
 
 
+
+def test_arg_with_choices():
+    xml_file = \
+        """\
+        <launch>
+            <arg name="my_arg" default="asd" description="something">
+                <choice value="asd"/>
+                <choice value="bsd"/>
+            </arg>
+        </launch>
+        """
+    xml_file = textwrap.dedent(xml_file)
+    root_entity, parser = Parser.load(io.StringIO(xml_file))
+    ld = parser.parse_description(root_entity)
+    arg = ld.entities[0]
+    assert 'my_arg' == arg.name
+    assert 'asd' == ''.join([x.perform(None) for x in arg.default_value])
+    assert "something Valid choices are: ['asd', 'bsd']" == arg.description
```

Comment:
@ivanpauno nit^N: I know this is pre-existing code, but could we ensure there's a period between the description and the list of choices:
```suggestion
    assert "something. Valid choices are: ['asd', 'bsd']" == arg.description
```
?

Reply:
the test doesn't path with that change, I would need to change the source code for that

Reply:
Right, that's what I meant by ensuring there's a period :sweat_smile: 

Sorry for the noisy comms.

Reply:
> Right, that's what I meant by ensuring there's a period sweat_smile

Sorry, I didn't read the first part of your original comment, I only saw the suggestion.
Fixed in https://github.com/ros2/launch/pull/529/commits/5d860a8693aed53ee80b0ec63cb45a8ffbd005ec